platform/windows/os_windows.cpp

```diff
@@ -773,7 +773,7 @@ LRESULT OS_Windows::WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
 				bmi.bmiHeader.biPlanes = 1;
 				bmi.bmiHeader.biBitCount = 32;
 				bmi.bmiHeader.biCompression = BI_RGB;
-				bmi.bmiHeader.biSizeImage = dib_size.x, dib_size.y * 4;
```

Comment:
I'm not exactly sure what the original intentions are with this comma operator, but I'm pretty sure my fix is good. The size of an image in bytes is height times width times 4 bytes per pixel.